automation/jleechanorg_pr_automation/tests/test_packaging_integration.py

```
@@ -0,0 +1,127 @@
+class TestPackageStructure:
+    """Verify overall package structure is correct."""
+
+    def test_all_package_dirs_have_init(self):
+        """
+        Verify all package directories have __init__.py files.
+
+        This prevents future packaging bugs.
+        """
+        # Imports moved to top level
+
+        package_root = os.path.dirname(jleechanorg_pr_automation.__file__)
```

Ruff (0.14.8)

`os.path.dirname()` should be replaced by `Path.parent`

(PTH120)